### PR TITLE
Parallel insert

### PR DESCRIPTION
**Pull request by dday01**
Changes to enable running the insert in parallel session via pg_cron.

### CODE REVIEW

File: BuildScripts/mvApplicationFunctions.sql [hunk 1]
```diff
@@ -152,6 +167,17 @@ DECLARE
 BEGIN
 
     rConst      := mv$buildAllConstants();
+	
+	IF pParallel = 'Y' THEN
+	
+		IF pParallelColumn IS NULL AND pParallelColumn IS NULL AND pParallelJobs = 0 THEN
```
Comment (ramiachouri) [inline BuildScripts/mvApplicationFunctions.sql]: IF pParallelColumn IS NULL OR **pParallelAlias** IS NULL OR pParallelJobs = 0 THEN

File: BuildScripts/mvSimpleFunctions.sql [hunk 2]
```diff
@@ -2217,6 +2327,8 @@ Revision History    Push Down List
 ------------------------------------------------------------------------------------------------------------------------------------
 Date        | Name          | Description
 ------------+---------------+-------------------------------------------------------------------------------------------------------
+23/07/2020	| D Day			| Added parallel running logic into truncate as this needs committing in the background due to locking
```
Comment (ramiachouri) [inline BuildScripts/mvSimpleFunctions.sql]: 23/07/2021

Comment (dday01) [reply BuildScripts/mvSimpleFunctions.sql]: corrected

File: BuildScripts/mvApplicationFunctions.sql (hunk 1 above)
Comment (dday01) [reply BuildScripts/mvApplicationFunctions.sql]: good spot - changed.

**Pull request closed without merging**